### ISSUE TITLE
The pileup view should pop out into a new window

### ISSUE THREAD

**Issue by jaclynperrone** [hammerlab/cycledash#687]
...and update its position as the user clicks on different variants in the Examine page.

This way the user has a little breathing room to read and filter the VCF file as they see fit. I imagine this new window to behave like the browser inspector, but we can hash out it's features elsewhere.

**Comment by hammer** [comment]
What about having a fixed position within the same window that is hidden when not in use? I feel like pop out windows are not a common design pattern on the web these days.

**Comment by ihodes** [comment]
I think that the pop-out pileup view is definitely worth exploring. I'm not convinced it's the right way to go, but I tend to think it'd be an improvement. 

Our current setup (which essentially is in a fixed position & hidden when not in use) eats of the majority of screen space, and skipping between pileups is difficult. There are definitely improvements we can make with the in-screen pileup viewer, though. 

I am okay with going against the grain on the journey to variant inspection nirvana.

**Comment by jaclynperrone** [comment]
I agree with @ihodes. I think both options are worth exploring, improving, and validating. As far as design patterns go, I think that the browser inspector serves a similar purpose to what we are doing here. And mind you, that starts out in browser and has the option to pop out in a new window. So maybe that functionality is something worth exploring.

**Comment by hammer** [comment]
The browser inspector is exactly what I had in mind. Making popping out an option seems reasonable; making it the default behavior seems odd.

**Comment by jaclynperrone** [comment]
Totally. I imagine popping it out on first click will lead to some confusion and loss of context. The cool thing about the browser inspector is that once you pop it out the first time, it stays that way until change your preference. Another behavior worth exploring!

**Comment by jaclynperrone** [comment]
Looping @danvk in! Here's one of the issues we talked about today.

**Comment by danvk** [comment]
@jaclynperrone and my thinking was that when you click "Open Pileup Viewer", it will show a full-page pileup viewer, with some information about the selected row on the top. There will be a button to "pop out" the pileup viewer into a separate window. This setting can be remembered via cookie.

If you click a row on the examine page while the pileup viewer is popped out, it will jump to that variant.

When you close the full-page pileup viewer (by hitting the "x" icon), the variant you were looking at will remain selected. This will facilitate the Inspect at a variant → Comment on that variant workflow.

**Comment by ihodes** [comment]
That sounds awesome, glad to see the popping out functionally revisited!

**Comment by jaclynperrone** [comment]
@danvk +1 +1 +100
